src/sentry/static/sentry/app/views/groupDetails/userWidget.jsx

```diff
@@ -1,6 +1,25 @@
+import _ from "underscore";
 import React from "react";
 import Gravatar from "../../components/gravatar";
 
+function keyToName(key) {
+  // Take a given key, and transform it from
+  // camel case to title case
+  if (!key) return '';
+  key = key[0].toUpperCase() + key.slice(1);
+  return key.replace(/_/g, ' ');
+}
+
+function renderLine(key, value, name='') {
+  if (name === '') {
+    name = keyToName(key);
+  }
+  return [
+    <dt key={key + "-label"}>{name}:</dt>,
+    <dd key={key}>{value.toString()}</dd>
```

Comment:
It wasn't behaving correctly when value was a boolean, fwiw. Which is why I added this.

Lemme dubble check.